background.js

```diff
@@ -62,18 +62,18 @@ chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
 			&& sender.tab.url == request.matchUrl) {
 				updateIcon(sender.tab, styles);
 			}
-			return true;
+			return KEEP_CHANNEL_OPEN;
 		case "saveStyle":
-			saveStyle(request, sendResponse);
-			return true;
+			saveStyle(request).then(sendResponse);
```

Comment:
Why would KEEP_CHANNEL_OPEN be false at any point? It's a constant that is equal to `true` defined in a script that loads before background.js. The entire reason for this constant is to have a self-explanatory name to indicate what this `return` does.